### ISSUE TITLE
footer.scss file isn't updated to include new $footer-padding variable

### ISSUE THREAD

**Issue by shereenqumsieh** [joshuajansen/bulma-rails#41]
I believe this was added on Oct 28th to bulma, but it isn't included in the official bulma 0.7.2 release notes. I'm trying to figure out why I see this in their documentation but it isn't working when I set it. I took a look at the bulma-rails gem and notice that the footer.scss file is actually missing the $footer-padding variable.

My apologies if there is confusion on my part, any help would be appreciated.

Basically, i'm trying to set $footer-variable and it isn't working.

**Comment by erikdahlstrand** [comment]
Hi,

Hopefully that's fixed in 0.7.3.
